services/movie_session.py

```diff
@@ -41,4 +45,12 @@
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
-    MovieSession.objects.get(id=session_id).delete()
+    get_object_or_404(MovieSession, id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
```

Comment:
Consider using `List[Dict[str, int]]` from the `typing` module for the return type annotation to improve type clarity.